### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RockyWearsAHat**

### CODE REVIEW

**Comment by RockyWearsAHat** [comment]
This is fine

**Review by RockyWearsAHat — COMMENTED** [review]
Good

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 So far this is a simple base framework that creates a new deck if there isn't a deck id found yet in the browser session, allows shuffling deck to reset game, drawing of cards for a specific user, and getting of hand for a particular user.
 
+<a href="https://deckofcardsapi.com/?ref=hackernoon.com">API Reference</a>
+
```
Comment (RockyWearsAHat) [inline README.md:6]: res

**Review by RockyWearsAHat — COMMENTED** [review]
This is good

**Review by Snowbirdlam — APPROVED** [review]
Good to go

**Pull request merged**